content/en/continuous_integration/setup_tests/dotnet.md

```diff
@@ -71,56 +71,24 @@ dd-trace --help
 
 The following list shows the default values for key configuration settings:
 
-
-`--set-ci`
-: Sets up the clr profiler environment variables for all the CI pipeline. <br/>
-**Default**: `false`
-
-`--dd-env`
-: Environment name for unified service tagging.<br/>
-**Environment variable**: `DD_ENV`<br/>
-**Default**: `(empty)`
-
 `--dd-service`
-: Service name for unified service tagging. <br/>
+: Name of the service or library under test.<br/>
 **Environment variable**: `DD_SERVICE`<br/>
-**Default**: `(ProcessName)` 
+**Default**: `(ProcessName)`<br/>
```

Comment:
In .NET the default is the repo name (https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/Common.cs#L43)